src/chapters/configuring.xml

```diff
@@ -3009,13 +3009,102 @@ ed272546-87bd-4db9-acba-e36e1a9ca20a
                                             <primary>SSH</primary>
                                             <secondary>color scheme</secondary>
                                         </indexterm>The color scheme to use for the terminal
-                                        emulator used by SSH connections. Each color scheme dictates
-                                        the default foreground and background color for the
-                                        terminal. Programs which specify colors when printing text
-                                        will override these defaults.</para>
-                                    <para>This parameter is optional. By default, Guacamole will
-                                        render text as gray over a black background.</para>
-                                    <para>Possible values are:</para>
+                                        emulator used by SSH connections. It consists of a
+                                        semicolon-separated series of name-value pairs, separated by
+                                        a colon, that assign default values to colors in the
+                                        terminal emulator palette. Note that programs that specify
+                                        colors when printing text can still override these defaults.
+                                        For example, to use blue text on white background by
+                                        default, and change the default red color to a purple shade,
+                                        you would specify:</para>
+                                    <informalexample>
+                                        <programlisting>foreground: rgb:00/00/ff;
+background: rgb:ff/ff/ff;
+color9: rgb:80/00/80</programlisting>
+                                    </informalexample>
+                                    <para>This format is similar to the color configuration format
+                                        used by Xterm, so Xterm color configurations can be easily
+                                        adapted for Guacamole. This parameter is optional. If not
```

Comment:
How does that "adaption" work?  Is there anything special that needs to be done when translating from Xterm color configurations to Guacamole?  If so, might make sense to put another `<informalexample>` block in right here.

Reply:
It goes from something like

    xterm*color9: rgb:cf/61/71
    xterm*color10: rgb:7c/bc/8c

to

    color9: rgb:cf/61/71;
    color10: rgb:7c/bc/8c;

I only mentioned it as a hint that users can look for premade color schemes from xterm configs. I think the conversion process is straightforward and obvious enough that it's unnecessary to include more details/examples.